src/test/cloudWatchLogs/registry/logDataRegistry.test.ts

```diff
@@ -50,6 +86,119 @@ describe('LogDataRegistry', async function () {
         })
     })
 
+    describe('updateLog', async function () {
+        it("properply paginates the results with 'head'", async () => {
+            await testUpdateLog('head')
+        })
+
+        it("properply paginates the results with 'tail'", async () => {
+            await testUpdateLog('tail')
+        })
+
+        function createCwlEvents(id: string, count: number): FilteredLogEvents {
```

Comment:
I don't think we have a standard on this, but I think we should pick where we define helper functions in describe blocks

Reply:
My thought process is that these functions are currently only used by the one test. So it makes it easier if they are all grouped together. But in the event another test can use them we can change the scope of the functions.

Reply:
Function declaration order doesn't matter as far as scoping is concerned. All functions declared in this block are usable by all tests. I usually put functions at the top of a block below any variables. In this situation I'd probably make a new nested describe block to put all the pagination stuff in.

Reply:
I've updated this test to its own describe block. Thanks!